test/LaunchDarkly.OpenFeature.ServerProvider.Tests/ProviderTests.cs

```diff
@@ -252,5 +270,39 @@ public void ItCanDoAValueEvaluation()
             var res = provider.ResolveStructureValue("flag-key", new Value("false"), evaluationContext).Result;
             Assert.Equal("true", res.Value.AsString);
         }
+
+        [Fact]
+        public async Task ItEmitsConfigurationChangedEvents()
+        {
+            var testData = TestData.DataSource();
+            var config = Configuration.Builder("")
+                .DataSource(testData)
+                .Events(Components.NoEvents)
+                .Build();
+
+            var provider = new Provider(config);
+            await provider.Initialize(EvaluationContext.Empty);
+
+            testData.Update(testData.Flag("test-flag-a").BooleanFlag().On(true));
+            testData.Update(testData.Flag("test-flag-b").BooleanFlag().On(true));
+
+            // Get the ready event.
+            await provider.GetEventChannel().Reader.ReadAsync();
+
+            // The ordering of the subsequent events is not going to be deterministic.
```

Comment:
As another PR I will investigate if I can order these. I need to look into the SDK code to understand how it runs these handlers.